pkgs/development/compilers/osl/default.nix

```diff
@@ -1,34 +1,44 @@
 { clangStdenv, lib, fetchFromGitHub, cmake, zlib, openexr,
-openimageio, llvm, boost165, flex, bison, partio, pugixml,
-util-linux, python
+openimageio2, llvmPackages, boost165, flex, bison, partio, pugixml,
+util-linux, python3
 }:
 
 let boost_static = boost165.override { enableStatic = true; };
 in clangStdenv.mkDerivation rec {
```

Comment:
Can we move this into top-level to make sure to only use this boost across the package?